socorro/signature/siglists/irrelevant_signature_re.txt

```diff
@@ -162,6 +162,8 @@ mozilla::ThreadSafeAutoRefCnt
 mozilla::UniquePtr
 mozilla::widget::WlCrashHandler
 mozilla::wr::RenderThread::AddRef
+~nsCOMPtr
+~nsCOMPtr_base
```

Comment:
These show up in the signature like `nsCOMPtr_base::~nsCOMPtr_base`, so these two lines have no effect on signature generation.

Further, `nsCOMPtr` is in the prefix list so signature generation already covers those frames.